cpp/test/Slice/errorDetection/IdentAsKeyword.err

```diff
@@ -56,14 +56,15 @@ IdentAsKeyword.ice:54: keyword 'idempotent' cannot be used as a name
 IdentAsKeyword.ice:54: enums can only be defined within a module
 IdentAsKeyword.ice:55: enums can only be defined within a module
 IdentAsKeyword.ice:55: enumeration 'IDEMPOTENT' differs only in capitalization from enumeration 'idempotent'
-IdentAsKeyword.ice:55: 'IDEMPOTENT' has changed meaning
 IdentAsKeyword.ice:57: enums can only be defined within a module
 IdentAsKeyword.ice:57: keyword 'long' cannot be used as enumerator
 IdentAsKeyword.ice:57: keyword 'byte' cannot be used as enumerator
 IdentAsKeyword.ice:58: enums can only be defined within a module
 IdentAsKeyword.ice:60: interfaces can only be defined within a module
+IdentAsKeyword.ice:60: keyword 'module' cannot be used as a name
```

Comment:
Here it's actually good to have 2 errors, because they're both true and unrelated.
We didn't see the 2nd error before, because the `createInterface` logic exited early with `nullptr`.
Now that it doesn't exit early, we report all the errors we see.